<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EbsdMontageImportWidget</class>
 <widget class="QFrame" name="EbsdMontageImportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>525</height>
   </rect>
  </property>
  <property name="title" stdset="0">
   <string>Image Stack Import</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="_6">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Generated Input File Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_24">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>12</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="generatedFileNameExample">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>[Prefix][Zero Padding]r[Row]c[Col][Suffix].[extension]</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_25">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QListWidget" name="fileListView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The list of files that will be imported in the order they will be imported into the HDF5 Ebsd File</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_57">
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>File List (Green=File Exists  Red=File Does NOT Exist)</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="errorMessage">
     <property name="text">
      <string>Error Messages</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="widgetLayout">
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="3" column="1">
      <widget class="QLabel" name="totalSlices">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_29">
       <property name="toolTip">
        <string>The directory that contains the .ang or .ctf files</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Input Directory:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel 
{
	font-style: italic;font-weight: bold;
font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>Files Source Directory</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QtSLineEdit" name="inputDir"/>
     </item>
     <item row="2" column="1">
      <widget class="SVSmallLabel" name="absPathLabel">
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="inputDirBtn">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_47">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Total Files Found:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="Line" name="line_9">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_58">
     <property name="styleSheet">
      <string notr="true">QLabel 
{
	font-style: italic;font-weight: bold;
font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>File Name Generation</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="gridLayout_20">
     <item row="1" column="3">
      <widget class="SVSpinBox" name="rowEnd">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The file index to stop the import at. (Inclusive)</string>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_50">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Row Start</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="SVSpinBox" name="rowStart">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Which file index to start the import at.</string>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_51">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Row End</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_56">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>File Extension:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="SVLineEdit" name="fileSuffix">
       <property name="toolTip">
        <string>The part of the filename that comes AFTER the slice index</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="SVLineEdit" name="fileExt">
       <property name="toolTip">
        <string>The file extension of the input files</string>
       </property>
       <property name="text">
        <string>ang</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_32">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>File Prefix:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="SVLineEdit" name="filePrefix">
       <property name="toolTip">
        <string>The front part of the input file name</string>
       </property>
       <property name="text">
        <string>Slice_</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>File Suffix:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_52">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Increment Index:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="SVSpinBox" name="increment">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Which file index to start the import at.</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_53">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Col Start</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="SVSpinBox" name="colStart">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Which file index to start the import at.</string>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_54">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Col End</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="SVSpinBox" name="colEnd">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The file index to stop the import at. (Inclusive)</string>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_21">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Num. Index Digits:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="SVSpinBox" name="totalDigits">
       <property name="toolTip">
        <string>The number of digits to allow for the slice index.</string>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SVLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">SVControlWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>QtSLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtSLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>SVSmallLabel</class>
   <extends>QLabel</extends>
   <header location="global">SVControlWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>SVSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">SVControlWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>inputDir</tabstop>
  <tabstop>inputDirBtn</tabstop>
  <tabstop>filePrefix</tabstop>
  <tabstop>fileSuffix</tabstop>
  <tabstop>fileExt</tabstop>
  <tabstop>rowStart</tabstop>
  <tabstop>rowEnd</tabstop>
  <tabstop>fileListView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
